debian/configure

```diff
@@ -89,8 +86,8 @@ PYTHON_IMAGING=python3-pil
 PYTHON_IMAGING_TK=python3-pil.imagetk
 QTVCP_DEPENDS="python3-pyqt5,\n    python3-pyqt5.qsci,\n    python3-pyqt5.qtsvg,\n    python3-pyqt5.qtopengl,\n    python3-opencv,\n    python3-dbus,python3-espeak,\n    python3-dbus.mainloop.pyqt5,\n    python3-pyqt5.qtwebkit,\n    espeak-ng,\n    pyqt5-dev-tools,\n    gstreamer1.0-tools,espeak,\n    sound-theme-freedesktop"
 YAPPS_RUNTIME="python3-yapps"
-DEBHELPER="debhelper (>= 9)"
-COMPAT="9"
+DEBHELPER="debhelper (>= 12)"
+COMPAT="12"
```

Comment:
If the new debhelper level should be 12, the --parallel line in debian/rules.in should be dropped.

Reply:
The oldest distro that master (2.9) supports is buster, which has [debhelper 12](https://packages.debian.org/buster/debhelper), so I'll remove the `--parallel`.